sopel/plugins/rules.py

```diff
@@ -982,10 +982,7 @@ def match_preconditions(self, bot, pretrigger):
         intent = pretrigger.tags.get('intent')
         nick = pretrigger.nick
         is_bot_message = (
-            (
-                'draft/bot' in pretrigger.tags or  # can be removed...someday
-                'bot' in pretrigger.tags
-            ) and
+            'bot' in pretrigger.tags and
             event in ["PRIVMSG", "NOTICE"]
         )
```

Comment:
```suggestion
        is_bot_message = 'bot' in pretrigger.tags and event in ["PRIVMSG", "NOTICE"]
```
Github has made this confusing but yeah the whole conditional fits in 85 chars now

Reply:
Oh, hmm. I was going for "don't touch the pieces that aren't changing", but let's see what @Exirel thinks about the style.

Reply:
85 is bigger than 79.